docs/guide/vue.md

```diff
@@ -0,0 +1,79 @@
+# Vue
```

Comment:
As discussed during the weekly meeeting, please add a banner/alert stating that support for vue is in its infancy